sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,287 @@
+UPDATE `areatrigger` SET `spawnDifficulties` = '1' WHERE `MapId` = 1331;
+UPDATE `areatrigger` SET `spawnDifficulties` = '1' WHERE `MapId` = 1358;
+UPDATE `areatrigger` SET `spawnDifficulties` = '14,15,16,17' WHERE `MapId` = 1448;
+UPDATE `areatrigger` SET `spawnDifficulties` = '1,2,8,23' WHERE `MapId` = 1456;
+UPDATE `areatrigger` SET `spawnDifficulties` = '23' WHERE `MapId` = 1456;
+UPDATE `areatrigger` SET `spawnDifficulties` = '1,2,8,23' WHERE `MapId` = 1458;
+UPDATE `areatrigger` SET `spawnDifficulties` = '12' WHERE `MapId` = 1460;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1463;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1464;
+UPDATE `areatrigger` SET `spawnDifficulties` = '1,2,8,23' WHERE `MapId` = 1466;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1468;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1469;
+UPDATE `areatrigger` SET `spawnDifficulties` = '12' WHERE `MapId` = 1475;
+UPDATE `areatrigger` SET `spawnDifficulties` = '1,2,8,23' WHERE `MapId` = 1477;
+UPDATE `areatrigger` SET `spawnDifficulties` = '23' WHERE `MapId` = 1477;
+UPDATE `areatrigger` SET `spawnDifficulties` = '12' WHERE `MapId` = 1478;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1479;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1481;
+UPDATE `areatrigger` SET `spawnDifficulties` = '1,2,8,23' WHERE `MapId` = 1492;
+UPDATE `areatrigger` SET `spawnDifficulties` = '1, 2, 8, 23' WHERE `MapId` = 1493;
+UPDATE `areatrigger` SET `spawnDifficulties` = '1,2,8,23' WHERE `MapId` = 1501;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1502;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1512;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1513;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1514;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1515;
+UPDATE `areatrigger` SET `spawnDifficulties` = '2,8,23' WHERE `MapId` = 1516;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1519;
+UPDATE `areatrigger` SET `spawnDifficulties` = '14,15,16,17' WHERE `MapId` = 1520;
+UPDATE `areatrigger` SET `spawnDifficulties` = '14,15,16,17' WHERE `MapId` = 1530;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1540;
+UPDATE `areatrigger` SET `spawnDifficulties` = '1,2,23' WHERE `MapId` = 1544;
+UPDATE `areatrigger` SET `spawnDifficulties` = '2,8,23' WHERE `MapId` = 1571;
+UPDATE `areatrigger` SET `spawnDifficulties` = '1,2,8,23' WHERE `MapId` = 1594;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1637;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1642;
+UPDATE `areatrigger` SET `spawnDifficulties` = '0' WHERE `MapId` = 1643;
+UPDATE `areatrigger` SET `spawnDifficulties` = '14, 15, 16, 17' WHERE `MapId` = 1648;
```

Comment:
also some lines with spaces in the middle

Reply:
Done